.github/workflows/slow_ci_remainder.yml

```diff
@@ -0,0 +1,46 @@
+name: Slow CI Reminder
+
+# For security reason, don't use `pull_request` but `pull_request_target`!
+on:
+  pull_request_target:
```

Comment:
For security reason, see [https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_target]

> This event runs in the context of the base of the pull request, rather than in the context of the merge commit, as the pull_request event does. This prevents execution of unsafe code from the head of the pull request that could alter your repository or steal any secrets you use in your workflow. This event allows your workflow to do things like label or comment on pull requests from forks. Avoid using this event if you need to build or run code from the pull request.

Reply:
Also we need to use `pull_request_target` so every time a push to a PR will trigger the workflow without an approval by internal team members

> Note: Workflows triggered by pull_request_target events are run in the context of the base branch. Since the base branch is considered trusted, **workflows triggered by these events will always run, regardless of approval settings**. For more information about the pull_request_target event, see "[Events that trigger workflows](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target)."